src/posts/donate.md

```diff
@@ -1,39 +1,40 @@
 
 <div style="display: flex; justify-content: center;" ><a href="https://www.paypal.com/donate/?hosted_button_id=4TWZXY62EM5VE"><img src="/PayPal.svg" alt="Donate now" width="256" /></a></div>
 
-## How we use your donations
+## What PauseAI has achieved in the past two years (check -- how long has it been?) 
+
+- 15 major [protests](/protests) in 7 countries with significant news coverage, with more to come
+- Significant [media appearances](/press) raising awareness of AI risks on major news outlets, documentaries, TV, podcasts and more
+- Attendance and participation in panels at __ AI safety summits
+- Dozens of well-attended presentations raising awareness about AI risks and how to lobby for a global pause on frontier AI research
+- Tens of thousands of volunteer hours spent on activism, flyering, local organizing, protesting, cold-calling, raising awareness, lobbying and other actitivies to help pause AI 
+- Hundreds of meetings lobbying and partnering with politicians, the wider AI safety community, and influential people (influencers, researchers, media, etc.) to work on the cause of stopping unsafe AI
+- A global community of volunteers ([join here](/join)) working around the clock to pause unsafe AI! 
 
-- **People**: build a stronger community, work on projects, organize protests, improve the website, etc.
-- **Events**: renting a location, catering, travel costs.
-- **Software**: email, website, etc.
-- **Campaigns**: online advertising, materials
-- **Legal & financial**: setting up the foundation (notary costs), bookkeeping.
-- **Community projects**: our [microgrants](/microgrants) program supports various small-scale projects.
 
-## What have we already achieved
 
-- 1000+ people who have [joined](/join) PauseAI, many of whom are actively working on various projects
-- 100+ projects in our Discord server (websites, workshops, videos, responses to government consultations, etc.)
-- Organized 15+ [protests](/protests) in 7 countries.
-- 30+ [Media appearances](/press) on major news outlets, podcasts, documentaries, TV and more.
-- Many meetings with politicians and other stakeholders.
+## What your donation helps to achieve 
 
-## What we're planning to do
+- Protests, summits, talks, and in-person events
+- Awareness campaigns, including online, media, and in-person
+- [Microgrants](/microgrants) supporting high-ROI small-scale projects around the world to help PauseAI
+- Contracting with professionals as needed to elevate our social media, video, branding, and media reach 
+- Tools to keep our volunteer organization running at high levels of effectivness (email, website, CRM, bookkeeping, project tools) 
+- PauseAI is a [registered non-profit foundation](/legal) in the US, UK, France, and the Netherlands.
 
-See our [roadmap](/roadmap) for more details.
+## Other ways you can become a donor
```

Comment:
Why remove the roadmap link? Seems good to include these details, especially for larger donations.